deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/assertions/Assertions.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.openapi.generator.deployment.assertions;
+
+import com.github.javaparser.ast.body.MethodDeclaration;
+
+import io.quarkiverse.openapi.generator.testutils.circuitbreaker.assertions.CircuitBreakerMethodAssert;
+
+public final class Assertions extends org.assertj.core.api.Assertions {
+
+    private Assertions() {
+    }
+
+    public static CircuitBreakerMethodAssert assertThat(MethodDeclaration actual) {
```

Comment:
Do you need that class? Why not call directly the references classs?

Reply:
Without that class, we would have to qualify the calls to `CircuitBreakerMethodAssert#assertThat`, since we wouldn't be able to statically import `assertThat` from `CircuitBreakerMethodAssert` and from `org.assertj.core.api.Assertions` at the same type.

Without that class the code would be like the following:

````java
import static org.assertj.core.api.Assertions.assertThat;
import io.quarkiverse.openapi.generator.testutils.circuitbreaker.assertions.CircuitBreakerMethodAssert;

@Test
void test() {
        // ...   

        // pure AssertJ assertion
        assertThat(byeMethod).isNotEmpty();

        // custom assertion
        CircuitBreakerMethodAssert.assertThat(byeMethod.orElseThrow())
                .hasCircuitBreakerAnnotation()
                .doesNotHaveAnyCircuitBreakerAttribute();

        // ...   
}
````

With that class the code would be like the following:

````java
// Our custom class includes the pure AsserJ assertions
import static io.quarkiverse.openapi.generator.deployment.assertions.Assertions.assertThat;

@Test
void test() {
        // ...   

        // pure AssertJ assertion
        assertThat(byeMethod).isNotEmpty();

        // custom assertion
        assertThat(byeMethod.orElseThrow())
                .hasCircuitBreakerAnnotation()
                .doesNotHaveAnyCircuitBreakerAttribute();

        // ...   
}
````

But, I can remove it if you think it's better.

Reply:
Thanks for the explanation. Keep it in the way you did.